stallman/vida-pessoal.md

```diff
@@ -1,4 +1,44 @@
 [Voltar](intro.md)
 
 Vida pessoal
-====
\ No newline at end of file
+====
+
+- Onde e quando nasceu? <br>
```

Comment:
Olá, @thiagoluigi7.

O texto de descrição do PR está ótimo. Só adicione, por favor, o nome dos outros integrantes da equipe, para eu poder dar nota para todos.

Quanto ao conteúdo, ficou ótimo. Está muito bem escrito e consistente com o que foi planejado! Sugiro apenas algumas pequenas modificações:

1. No título das seções, usar `##` ao invés de um `-`. Tipo assim: `## Onde e quanto nasceu?`
2. Dar um espaço entre o final da palavra e a nota de rodapé, tipo assim: `Estados Unidos [^1]`.
3. Para a citação que o Stallman faz sobre o dinheiro, usar o recurso de citação do Markdown, conforme descrito [aqui](https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-text).
  
Obrigado pela contribuição de vocês.

Atenciosamente.
Paulo.

Reply:
Feito 👍🏻